ssc/libssc.c

```diff
@@ -1158,15 +1819,8 @@ static PetscErrorCode PCView_PATCH(PC pc, PetscViewer viewer)
         ierr = PetscViewerASCIIPrintf(viewer, "Saving patch operators (rebuilt every PCSetUp)\n"); CHKERRQ(ierr);
     }
     ierr = PetscViewerASCIIPrintf(viewer, "DM used to define patches:\n"); CHKERRQ(ierr);
-    ierr = PetscViewerASCIIPushTab(viewer); CHKERRQ(ierr);
```

Comment:
Should revisit this to give all necessary information.  E.g. it's not only additive, or vertex patch, etc...

Reply:
I've added some more info in the view method.